conda_recipe_manager/parser/recipe_parser_convert.py

```diff
@@ -738,6 +740,16 @@ def pre_process_recipe_text(content: str) -> str:
                     f"env.get({quote_char}{key}{quote_char})",
                 )
             )
+
+        for groups in cast(list[tuple[str, ...]], Regex.PRE_PROCESS_ENVIRON_GET.findall(content)):
+            key_quote_char, key, _, default_quote_char, default, _ = groups
```

Comment:
Arguably, I should use this expansion form over indexing in the other transformations :)